modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/graphql/servlet/GraphQLServletExtender.java

```diff
@@ -487,15 +496,39 @@ private boolean _equals(
 					}
 				}
 
-				if (childrenGraphQLType1.size() ==
-						childrenGraphQLType2.size()) {
+				if (childrenGraphQLFieldDefinitions1.size() ==
+						childrenGraphQLFieldDefinitions2.size()) {
 
 					return true;
 				}
 
 				return false;
 			}
 
+			private List<GraphQLFieldDefinition> _getGraphQLFieldDefinitions(
+				GraphQLType graphQLType1) {
```

Comment:
Are you sure that all that we want to compare are GraphQLFieldDefinitions?

Is this working for conflicts between full types like Creator or something similar that is supposed to be a GraphQLInputType?

Reply:
That's totally right. I focused on the GraphQLOutputType and wrongly consider that we will always have GraphQLFieldDefinitions.

I will consider other cases also, as the GraphQLInputObjectType and GraphQLEnumValueDefinition, and send a new solution